src/app/ApplicationTemplate.Shared.Views/Framework/Email/EmailService.Mobile.cs

```diff
@@ -0,0 +1,28 @@
+﻿//-:cnd:noEmit
+#if __ANDROID__ || __IOS__
+using System.Threading.Tasks;
+
+namespace ApplicationTemplate;
+
+public sealed partial class EmailService : IEmailService
+{
+	public async Task Compose(Email email)
+	{
+		var emailMessage = new Microsoft.Maui.ApplicationModel.Communication.EmailMessage()
+		{
+			Subject = email.Subject,
+			Body = email.Body,
+		};
+
+		foreach (var attachment in email.Attachments)
+		{
+			emailMessage.Attachments.Add(
+				new Microsoft.Maui.ApplicationModel.Communication.EmailAttachment(attachment.FullPath, attachment.ContentType)
```

Comment:
Can't we make usage of a using here on `Microsoft.Maui.ApplicationModel.Communication` ?

Reply:
May be, I just don't want any conflict between the Presentation layer's files, they have the same name.